skyglobal/tiles
# Add styling for Watch brand

We're adding a new brand called 'Watch' that needs its own styling.

**If you want to test:** 
- May have to run the branch `watch-brand` on the cms if Watch brand is not in master
- Edit a tile in local CMS with brand 'Watch' and add a channel to it
- start PPP with `CMS_SERVER=http://local.sky.com:3006 npm run start`
- Point pages `tiles` npm module at #watch-brand rather than latest version
- start pages with `npm run dev-with-ppp`
- find tile you edited and check styling is correct

**Screenshot:** 
![screen shot 2016-09-16 at 10 33 04](https://cloud.githubusercontent.com/assets/6567781/18581562/1ddbf3d8-7bf9-11e6-99be-7eb576c708cf.png)

**Changes in PR:**
- added dark grey (same as engagement tiles) as hover effect on watch branded tiles
- added black title text as hover effect on watch branded tiles
